### DISCUSSION TITLE
Progressive Web Apps in .NET 8

### DISCUSSION THREAD

**Discussion by Jimmys20** [dotnet/aspnetcore#50837] [General]
I have some ASP.NET Core hosted Blazor WASM applications that also use service workers so they can be installed as PWA. What is the recommended approach to move forward to .NET 8 now that wasm hosted template is removed? Can you create PWA using the new Blazor web app template?

**Comment by FM1973** [comment]
Good question.
I think you can still use the "old" WASM hosted template.
Regarding Blazor United I have no clue as well.

**Comment by YuJin44** [comment]
I kinda have the same question. I am not sure "united" template fits this scenario. You can still use the "old" template but not with dotnet8. I have stumbled across "dotnet aspire" online, which does have some similarities, but it is not accessible right now, only in preview. So this is strange to me why microsoft removed the hostet template while not providing a clear replacement/successor/alternative. I mean there are tutorials on the internet already how to manually "recreate" the "old" template with dotnet8 because there seems just nothing official...
